src/repositories/repository-factory.js

```diff
@@ -1,8 +1,21 @@
 'use strict';
 
+const Sequelize = require('sequelize');
+
+const { Op } = Sequelize;
+
 module.exports = function createRepository(model) {
-  function list() {
-    return model.findAll({});
+  function list(queryParams) {
+    const { limit, offset, contains } = queryParams;
+    return model.findAll({

```

Comment:
In order to have a proper pagination the result from server should contain the requested params also:

```
{
  total: // total elements,
  limit: // limit asked by the requester
  offset: // offset asked by requester
  data: [] // the list of results
}
```
```